internal/pkg/template/workload_test.go

```diff
@@ -479,3 +471,36 @@ func TestRollingUpdateRollbackConfig_TruncateAlarmName(t *testing.T) {
 		})
 	}
 }
+
+func TestApplicationLoadBalancer_Aliases(t *testing.T) {
```

Comment:
Since we now also have the `Aliases` method for NLB, do we need a unit test suite for it as well?

Reply:
Both the functions are same that's why I just kept tests for one. But happy to add for nlb as well.

Reply:
Yeah I think in this case we should. Ideally the tests are agnostic of the implementation: it doesn't know if the implementations are actually the same or not.